__tests__/__system__/api/methods/delete/Delete.webservice.system.test.ts

```diff
@@ -0,0 +1,101 @@
+/*
+* This program and the accompanying materials are made available under the terms of the *
+* Eclipse Public License v2.0 which accompanies this distribution, and is available at *
+* https://www.eclipse.org/legal/epl-v20.html                                      *
+*                                                                                 *
+* SPDX-License-Identifier: EPL-2.0                                                *
+*                                                                                 *
+* Copyright Contributors to the Zowe Project.                                     *
+*                                                                                 *
+*/
+
+import { Session } from "@zowe/imperative";
+import { ITestEnvironment } from "../../../../__src__/environment/doc/response/ITestEnvironment";
+import { TestEnvironment } from "../../../../__src__/environment/TestEnvironment";
+import { generateRandomAlphaNumericString } from "../../../../__src__/TestUtils";
+import { defineWebservice, deleteWebservice, IWebServiceParms } from "../../../../../src";
+
+let testEnvironment: ITestEnvironment;
+let regionName: string;
+let csdGroup: string;
+let session: Session;
+
+describe("CICS Delete web service", () => {
+
+    beforeAll(async () => {
+        testEnvironment = await TestEnvironment.setUp({
+            testName: "cics_cmci_delete_webservice",
+            installPlugin: true,
+            tempProfileTypes: ["cics"]
+        });
+        csdGroup = testEnvironment.systemTestProperties.cmci.csdGroup;
+        regionName = testEnvironment.systemTestProperties.cmci.regionName;
+        const cmciProperties = await testEnvironment.systemTestProperties.cmci;
+
+        session = new Session({
+            user: cmciProperties.user,
+            password: cmciProperties.password,
+            hostname: cmciProperties.host,
+            port: cmciProperties.port,
+            type: "basic",
+            strictSSL: false,
+            protocol: testEnvironment.systemTestProperties.cmci.protocol as any || "http",
+        });
+    });
+
+    afterAll(async () => {
+        await TestEnvironment.cleanUp(testEnvironment);
+    });
+
+    const options: IWebServiceParms = {
+        pipelineName: "AAAA1234",
+        wsBind: "/u/exampleapp/wsbind/example.log",
+        validation: false
+    } as any;
+
+    it("should delete a web service from CICS", async () => {
+        let error;
+        let response;
+
+        const webserviceNameSuffixLength = 4;
+        const webserviceName = "AAAA" + generateRandomAlphaNumericString(webserviceNameSuffixLength);
+
+        options.name = webserviceName;
+        options.csdGroup = csdGroup;
+        options.regionName = regionName;
+
+        try {
+            await defineWebservice(session, options);
+            response = await deleteWebservice(session, options);
```

Comment:
Sorry for missing this on other tests (for other commands)
If possible, could we list/view (`get resource`) the webservice that was just defined before deleting?

That being said though, why don't we just simplify system test scenarios a little bit or  maybe combine things like
- define and delete
- install and discard
- enable and disable
- OR a big one that does it all :smile:

Reply:
This is not a requirement for this PR